packages/jest-config/src/normalize.js

```diff
@@ -548,6 +549,24 @@ export default function normalize(options: InitialOptions, argv: Argv) {
       case 'testRegex':
         value = options[key] && replacePathSepForRegex(options[key]);
         break;
+      case 'moduleFileExtensions': {
+        value = options[key];
+
+        // If it's the wrong type, it can throw at a later time
```

Comment:
maybe we can mention about not being able to inject Jest into runtime here? 

From the PR summary:
>If js is missing from moduleFileExtension, jest is unable to inject into the runtime. I decided to throw an explicit configuration error rather than fixing jest-jasmine's require, as we'd also need all of our dependencies to do the same (e.g. source-map throws if we do moduleFileExtension: [] now).

Reply:
in the comment, or in the error message to the user?

Reply:
the comment in particular is about the `Array.isArray` check, if that wasn't obvious

Reply:
comment about type is fine. I was thinking about leaving a message why we care about "js" being always present